tests/integration/targets/win_contained_availability_group/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Var block
+  vars:
+    ag_name: "containedAG"
+    cluster_type: "None"
+    contained_availability_group: true
+    force: false
+  module_defaults:
+    lowlydba.sqlserver.availability_group:
+      sql_instance: "{{ sqlserver_instance }}"
+      sql_username: "{{ sqlserver_username }}"
+      sql_password: "{{ sqlserver_password }}"
+      ag_name: "{{ ag_name }}"
+      cluster_type: "{{ cluster_type }}"
+      failover_mode: "{{ failover_mode }}"
+      dtc_support_enabled: "{{ dtc_support_enabled }}"
+      availability_mode: "{{ availability_mode }}"
+      seeding_mode: "{{ seeding_mode }}"
+      database_health_trigger: "{{ database_health_trigger }}"
+      database: "{{ database }}"
```

Comment:
Most of these default values will need to be defined in the `vars` block above (see the `win_availability_group` test file for reference)